README.md

```diff
@@ -46,11 +46,12 @@ This project is a collection of [PHP_CodeSniffer](https://github.com/squizlabs/P
 
 ## Project history
 
- - In April 2009 original project from [Urban Giraffe](https://urbangiraffe.com/articles/wordpress-codesniffer-standard/) was published.
- - In May 2011 the project was forked on GitHub by [Chris Adams](https://chrisadams.me.uk/).
+ - On 22nd April 2009, the original project from [Urban Giraffe](https://urbangiraffe.com/articles/wordpress-codesniffer-standard/) was packaged and published.
```

Comment:
* What is the reason to switch from _month_ to _exact date_ ?
* Why "_packaged and published_" ? What does packaged mean in this case ?